jax/_src/scipy/special.py

```diff
@@ -38,10 +39,7 @@ def gammaln(x: ArrayLike) -> Array:
   return lax.lgamma(x)
 
 
-@_wraps(osp_special.betaln, module='scipy.special')
-def betaln(x: ArrayLike, y: ArrayLike) -> Array:
-  x, y = _promote_args_inexact("betaln", x, y)
-  return lax.lgamma(x) + lax.lgamma(y) - lax.lgamma(x + y)
+_wraps(betaln, module='scipy.special', update_doc=False)
```

Comment:
this should be something like
```python
betaln = _wraps(osp_special.betaln, module='scipy.special')(betaln)
```
As currently written, the output of `_wraps` is unassigned and so it will have no effect.

Reply:
Great catch, thanks